pydeps/render_context.py

```diff
@@ -27,13 +29,15 @@ def graph(self, **kw):
         self.name = kw.get('name', 'G')
         self.fillcolor = kw.get('fillcolor', '#ffffff')
         self.fontcolor = kw.get('fontcolor', '#000000')
+        self.rankdir = kw.get('rankdir', 'TB' if not self.reverse else 'BT')
```

Comment:
I appreciate that you're trying to kkep the non-reverse case the default, but this reads better if you reverse the logic 
```suggestion
        self.rankdir = kw.get('rankdir', 'BT' if self.reverse else 'TB')
```

Reply:
I also changed this, in addition there is now also a test for it.